vtex-apps/structured-data
# Uses endpoint to get settings

**What problem is this solving?**

This PR removes the apps-graphql dependency in favor of retrieving settings through the `getAppSettings` context method.

To achieve this, we needed to:
1) set up a node environment;
2) Create a graphQL endpoint to retrieve the settings using the context
3) Use this endpoint to retrieve the settings at the react app
4) Use the settings as usual

**How should this be manually tested?**

This app is responsible to build the structured-data of products, so it's not a visual change. But you can manually test by adding a console.log at the `useAppSettings` hook and check if the settings are being retrieved correctly.
